appion/wizards.py

```diff
@@ -35,7 +35,8 @@
 from pyworkflow.em import CoordinatesObjectView
 from pyworkflow.utils import makePath, cleanPath, readProperties
 
-from protocol_dogpicker import DogPickerProtPicking
+from protocols.protocol_dogpicker import DogPickerProtPicking
+from constants import DOGPICKER_HOME
```

Comment:
I guess this import will fail...
I will propose the same import convention:
from appion.constants import ....